### PR TITLE
Feature#10/styleviews

### LINKED ISSUE #10: Styles to views
# REQUIREMENTS
Add styles to views.

# Acceptance Criteria
List all required acceptance criteria
- [ ] Home for unregistered
- [ ] Login
- [ ] User profile
- [ ] User Registration
- [ ] List of complaints
- [ ] Complaint View
- [ ] Complaint form

### PR DESCRIPTION
**Pull request by brayandiazc**
Styles are applied to views using the Bulma CSS framework.

The modified views are:
- Login
- User Registration
- Change of password
- Resend confirmation instructions
- Complaint new
- Places of interest

# Design
complaint-new
interestPlaces-Index
interestPlaces-new
login
resentConfirmation
signup

Github issue # 19 (Admin panel) merge is expected to complete the remaining views

### CODE REVIEW

**Review by ALAN1MACIAS — DISMISSED** [review]

File: app/javascript/custom/navbar.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+        // Toggle the "is-active" class on both the "navbar-burger" and the "navbar-menu"
+        el.classList.toggle('is-active');
+        $target.classList.toggle('is-active');
+
+      });
+    });
+  }
+
+});
```
Comment (YilKanda) [inline app/javascript/custom/navbar.js:25]: Add a line blank

File: app/views/partials/_notification.html.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    <button class="delete"></button>
+    <%= alert %>
+  </div>
+<% end %>
+<% if flash[:error] %>
+  <div id="notice_wrapper" class="notification is-danger has-text-centered">
+    <button class="delete"></button>
+    <%= flash[:error] %>
+  </div>
+<% end %>
```
Comment (YilKanda) [inline app/views/partials/_notification.html.erb:18]: Add a line blank

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,45 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
```
Comment (YilKanda) [inline db/seeds.rb:8]: If you don't occupy these comments, you can delete them.

File: app/javascript/custom/navbar.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+document.addEventListener('turbolinks:load', () => {
```
Comment (YilKanda) [inline app/javascript/custom/navbar.js:1]: All right, use turbolinks.

**Review by YilKanda — COMMENTED** [review]
Just leave a few comments regarding the Style Guide, the rest looks good to me.

**Review by Carlos-Montiel — DISMISSED** [review]

File: app/javascript/custom/navbar.js (hunk 1 above)
Comment (brayandiazc) [reply app/javascript/custom/navbar.js:25]: ok, in the next PR they will be added.

File: app/views/partials/_notification.html.erb (hunk 2 above)
Comment (brayandiazc) [reply app/views/partials/_notification.html.erb:18]: because?

File: db/seeds.rb (hunk 3 above)
Comment (brayandiazc) [reply db/seeds.rb:8]: ok

**Review by ALAN1MACIAS — APPROVED** [review]

**Review by Carlos-Montiel — APPROVED** [review]

File: app/views/partials/_notification.html.erb (hunk 2 above)
Comment (YilKanda) [reply app/views/partials/_notification.html.erb:18]: This is what the Ruby Styles Guide indicates, and using console writing, line blank helps a lot.

**Pull request merged**